app/components/Request/messages.js

```diff
@@ -0,0 +1,22 @@
+import { defineMessages } from 'react-intl'
+
+/* eslint-disable max-len */
+export default defineMessages({
+  amount: 'Amount',
+  button_text: 'Request',
+  copy_Address: 'Copy Address',
```

Comment:
I think this should be `copy invoice` as we typically refer to onchain addresses as an "address"

Reply:
👍